### ISSUE TITLE
rpm scriplet contains typo

### ISSUE THREAD

**Issue by dada216** [gravitl/netclient#84]
```
rpm -qp --scripts netclient-0.17.0-0.x86_64.rpm
postinstall scriptlet (using /bin/sh):
systemctl preset netclient
systemctl daemon-reload
system**clt** enable netclient
systemctl start netclient
preuninstall scriptlet (using /bin/sh):
if [ "$1" = 0 ]; then
    if [ -x /sbin/netclient ]; then
        /sbin/netclient uninstall
    fi
fi
```

the line system**clt** enable netclient should read system**ctl** enable netclient

**Comment by mattkasun** [comment]
fixed in packaging repo.

**Comment by dbd** [comment]
Doesn't appear to be fixed in 0.18.5

```
rpm -qp --scripts netclient-0.18.5-0.x86_64.rpm
postinstall scriptlet (using /bin/sh):
systemctl preset netclient
systemctl daemon-reload
systemclt enable netclient
systemctl start netclient
```
